README.md

```diff
@@ -190,6 +248,45 @@ The HTML generated by this might look like the following:
 
 The `src` attribute will have [imgix URL API parameters](https://www.imgix.com/docs/reference) added to it to perform the resize.
 
+### Global Configuration
+
+The global configuration for this library should be located in `APP.imgix`, and has the following schema:
+
+```js
+imgix: {
+  source: string,
+  debug?: boolean,
+  attributeNameMap?: {
+    src: string
+    srcset: string
+    sizes: string
+  }
+}
+```
+
+It should look like this in `config/environment.js`.
+
+```js
+// config/environment.js
+
+module.exports = function(environment) {
+  var ENV = {
+    // snip
+    APP: {
+      imgix: {
+        source: 'my-social-network.imgix.net',
+        debug: true // Prints out diagnostic information on the image itself. Turn off in production.
+      }
+    }
+    // snip
+  };
+};
+```
+
+#### attributeNameMap
+
+`attributeNameMap` should be used if it is required to set the src of the image desired to another HTML attribute, rather than to
```

Comment:
I think the cliffhanger is a valuable narrative device if used effectively, but I'm not sure code documentation is the right place for it.

Reply:
😂😂🤔🙃